geometry_msgs/Point start3D
geometry_msgs/Point end3D
float32[4] hessian_left
float32[4] hessian_right
uint8 line_type
